app/controllers/admin/backups_controller.rb

```diff
@@ -44,9 +45,22 @@ def cancel
     render json: success_json
   end
 
-  # download
-  def show
+  def email
     if backup = Backup[params.fetch(:id)]
+      token = EmailBackupToken.set
+      hostname = Discourse.current_hostname
+      base_url = Discourse.base_url
+      download_url = "#{Discourse.base_url}/admin/backups/#{backup.filename}?token=#{token}"
```

Comment:
Are we able to generate the path from `admin_backups_path` instead of hardcoding the path here?